scripts/diagnostics.txt

```diff
@@ -693,6 +692,7 @@ warning event-const EventExpressionConstant "edge expression is constant"
 warning empty-stmt EmptyStatement "extra ';' has no effect"
 warning pointless-void-cast PointlessVoidCast "cast to void for void-returning function '{}' has no effect"
 warning unused-result UnusedResult "ignoring return value of '{}', cast to void to suppress"
+warning bad-procedural-force BadProceduralForce "lvalue of force/release must be a net, a variable, a constant select of a net, or a concatenation of these"
```

Comment:
Can you make sure this is included in the default set at the bottom of this file?